consul/acl.go

```diff
@@ -37,24 +37,29 @@ const (
 	redactedToken = "<hidden>"
 
 	// Maximum number of cached ACL entries
-	aclCacheSize = 256
+	aclCacheSize = 10 * 1024
```

Comment:
Can we use this chance to switch from the LRU cache to 2Q? It's in the same package!